README.md

```diff
@@ -14,27 +14,33 @@ In this project, we are setting up a "Hello world" repository with the use of on
 
 [Live Demo Link](http://127.0.0.1:5500/)
 
-
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
these instructions are for you to understand how to use the readme file. next time delete any instructions and it should look better.